contribute/contribution-guide.md

```diff
@@ -86,15 +120,15 @@ Clone Beam’s read-only GitHub mirror.
 
 Add your forked repository as an additional Git remote, where you’ll push your changes.
 
-<pre><code>$ git remote add <b>&lt;GitHub_user&gt;</b> git@github.com:<b>&lt;GitHub_user&gt;</b>/incubator-beam.git</code></pre>
+	$ git remote add <GitHub_user> git@github.com:<GitHub_user>/incubator-beam.git
```

Comment:
Good from what I can tell. The downside is we loose the ability to emphasize certain parts of the phrase, but at least we don't have stray html tags ;-)